hadoop-ozone/dist/src/main/compose/ozone/docker-config

```diff
@@ -20,14 +20,22 @@ OZONE-SITE.XML_ozone.scm.names=scm
 OZONE-SITE.XML_ozone.scm.datanode.id.dir=/data
 OZONE-SITE.XML_ozone.scm.block.client.address=scm
 OZONE-SITE.XML_ozone.metadata.dirs=/data/metadata
-OZONE-SITE.XML_ozone.handler.type=distributed
+OZONE-SITE.XML_ozone.recon.db.dir=/data/metadata/recon
+OZONE-SITE.XML_ozone.recon.om.db.dir=/data/metadata/recon
```

Comment:
/data/metadata/reon -> /data/metadata/om

Reply:
Thanks @bharatviswa504 for spotting this.  These config values from `ozone-recon` env as they were.

@avijayanhwx @swagle can Recon use the same directory for both `ozone.recon.db.dir` and `ozone.recon.om.db.dir`?

Reply:
@adoroszlai Yes the same directory can be used.

Reply:
Thanks @avijayanhwx.  Then I think we can keep it as is.